exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/data/data_customer_support.ipynb

```diff
@@ -0,0 +1,147 @@
+{
```

Comment:
<p>I would rather be vague about the data cleaning we've done. You may create an impression that from the original 8500 tickets you selected 3606 which the Text AI works for.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/364/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>